buildenv/jenkins/JenkinsfileBase

```diff
@@ -26,6 +75,30 @@ def test() {
 					CUSTOMIZED_SDK_URL = ''
 				}
 				env.SPEC = "${SPEC}"
+				env.BUILD_LIST = "${getBuildList()}"
+				echo "env.BUILD_LIST is ${env.BUILD_LIST}"
```

Comment:
This is redundant.  sh 'printenv' print all environmental variables.

Reply:
updated